src/mono/browser/runtime/exports-binding.ts

```diff
@@ -44,15 +44,15 @@ export const mono_wasm_threads_imports = !WasmEnableThreads ? [] : [
     mono_wasm_dump_threads,
 
     // corebindings.c
-    mono_wasm_install_js_worker_interop,
-    mono_wasm_uninstall_js_worker_interop,
-    mono_wasm_invoke_jsimport_MT,
-    mono_wasm_warn_about_blocking_wait,
+    SystemJSInterop_InstallWebWorkerInteropJs,
+    SystemJSInterop_UninstallWebWorkerInterop,
```

Comment:
Should both Install and Uninstall have the same suffix? (InteropJs vs. Interop)

If you keep `Js` in the name, it may want to be capitalized as `InteropJS` to match `SystemJS`.

Reply:
- `SystemJSInterop_InstallWebWorkerInterop` is C which wraps `SystemJSInterop_InstallWebWorkerInteropJs` 
- `SystemJSInterop_UninstallWebWorkerInterop` is mapped from JS directly to ICall. In a way, `SystemJSInterop_InstallWebWorkerInteropJs` is implementation detail.

Reply:
capitalized JS suffix